.travis.yml

```diff
@@ -5,6 +5,7 @@ env:
   - CMD="sass:compile"
   - CMD="lint:sass"
   - CMD="lint:js"
+  - CMD="test:karma"
```

Comment:
This is as good as a time as any to get more familiar with TravisCI headless testing facilities. It looks like Karma and Firefox together have known funny behavior in the Travis environment:

https://docs.travis-ci.com/user/gui-and-headless-browsers/#Karma-and-Firefox-inactivity-timeouts

Pivoting to PhantomJS is the quickest and easiest solution, but has some pros/cons:

* PhantomJS adds a heavy download to `npm install`, but it's self-contained
* We fracture consistent use of Firefox across repos, but it will work regardless and for everyone (including Travis)
* We will need to add a polyfill, but that will be transparent to devs

@WidgetsBurritos @xinbin thoughts?